jerry-core/ecma/base/ecma-helpers.c

```diff
@@ -937,7 +927,7 @@ ecma_delete_property (ecma_object_t *object_p, /**< object */
 
     for (int i = 0; i < ECMA_PROPERTY_PAIR_ITEM_COUNT; i++)
     {
-      if ((cur_prop_p->types + i) == prop_p)
+      if ((prop_pair_p->values + i) == prop_value_p)
```

Comment:
The point of the "prop_pair_p->values + i" form that it is the same as "&(prop_pair_p->values[i])" just looks better. I would rather keep it.